### PR TITLE
Update _TEMPLATE.md and update "please also refer to" section

### PR DESCRIPTION
**Pull request by Stuti-mishra**
Issue : Add "Related terms" section to template #792

### CODE REVIEW

**Review by iamNoah1 — CHANGES_REQUESTED** [review]
Thanks, @Stuti-mishra. Catherine asked to update the existing terms mentioned in the issue. Could you please add those changes to this PR as well? That would be awesome :)

**Comment by Stuti-mishra** [comment]
> Thanks, @Stuti-mishra. Catherine asked to update the existing terms mentioned in the issue. Could you please add those changes to this PR as well? That would be awesome :)

Hey! I'm sorry ! I've updated the existing terms here:
Update the existing terms.md #809

**Comment by Stuti-mishra** [comment]
> Thanks, @Stuti-mishra. Catherine asked to update the existing terms mentioned in the issue. Could you please add those changes to this PR as well? That would be awesome :)

Yes! Earlier I had created a separate pr , I've now made the foresaid changes and updated this one itself.
"Please refer to" section heading has been changed to "Related terms" for the following terms:
CI/CD and vertical/horizontal scaling/autoscaling

**Review by CathPag — APPROVED** [review]
Looks great, thank you!

File: content/en/_TEMPLATE.md [hunk 1]
```diff
@@ -5,11 +5,13 @@ category: concept
...
 ## Problem it addresses
-A few lines of the problem it's addressing.
+Define the problem it addresses. Ideally, don't even mention the term you are defining. 
 
 ## How it helps
-A few lines of how the thing solves the problem.
+Describe how the term addresses the problem described above.
 
+## Related terms
+Add related Glossary terms (if applicable)
```
Comment (jihoon-seo) [inline content/en/_TEMPLATE.md:17]: I suggest to insert empty lines before and after the header lines. 😊

- https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines
- https://arcticicestudio.github.io/styleguide-markdown/rules/headings.html#empty-lines-before-and-after

**Review by jihoon-seo — COMMENTED** [review]
@Stuti-mishra Thank you for your contribution. 😊
And PTAL!

File: content/en/_TEMPLATE.md (hunk 1 above)
Comment (Stuti-mishra) [reply content/en/_TEMPLATE.md:17]: Sure! I'll make those changes

Comment (Stuti-mishra) [reply content/en/_TEMPLATE.md:17]: lgtm!
let me know if I have to work on any changes

**Review by iamNoah1 — COMMENTED** [review]
I guess the suggestion is to actually add empty lines before and after headlines :)

**Comment by Stuti-mishra** [comment]
> I guess the suggestion is to actually add empty lines before and after headlines :)

Thanks for clarifying, I'm kinda new to this. 
I've made the changes, let me know if it looks good

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Review by iamNoah1 — APPROVED** [review]
All fine @Stuti-mishra, thanks for the contribution :)

**Pull request merged**